### DISCUSSION TITLE
[x][closed] Why are most accession numbers prepended with zeros?

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#86] [Team Posts]
I can only see downsides with this..

**Comment by sveinugu** [comment]
José María's answer: Fixed size is usually used to allow identifying broken or invalid identifiers, and even giving some meaning to some of their alphanumeric positions. But, they can also be a problem, as they usually require a centralized ticketing system, in order to record which is the next available one.

**Comment by sveinugu** [comment]
@jmfernandez I am not sure how fixed size is related to a centralized ticketing system. I suppose one would need such a ticketing system, even though one provides accession numbers in the form "fairtracks:1", "fairtracks:2" or "fairtracks:00000001", "fairtracks:00000002". The disadvantage of the last (and most often used) solution is that you will need to set a maximum number.

**Comment by dzerbino** [comment]
People complain about those in Ensembl, but because of historical code, we do not want to break existing parsers/writers. But yes, people hate them.
